pkg/tenant/db/errors.go

```diff
@@ -45,6 +45,7 @@ var (
 
 	// Max-length errors
 	ErrProjectDescriptionTooLong = errors.New("project description is too long")
+	ErrTopicNameTooLong          = errors.New("topic name is too long")
```

Comment:
Can you make sure the frontend is validating this as well or create a story for it?

Reply:
I'll double check the regex currently in use for validation on the FE side.